### PR TITLE
Simplify controller translation pages

### PR DESCRIPTION
**Pull request by oraNod**
Follows up on #105 and #106 to simplify the HTML pages for Controller translations and the docs archive.

### CODE REVIEW

**Comment by oraNod** [comment]
@BhattacharjeeSutapa Here is that PR I mentioned in the DaWGs meeting today for your FYI. The changes here improve the Jinja templating to simplify HTML files.

**Comment by oraNod** [comment]
Relates to issue #108

File: data/controller_i18n_links.yaml [hunk 1]
```diff
@@ -0,0 +1,509 @@
+# Translation links for Controller
+# Start Korean translations
+tower_ko:
```
Comment (samccann) [inline data/controller_i18n_links.yaml:3]: this is a total nit but if these are all controller pages, why do we use things like tower_ko etc throughout the new docsite? Maybe that's an easyfix issue we can create for someone else to come in and clean that up?

**Review by samccann — APPROVED** [review]
Verified locally that the docsite seems to look that same as what is published (aka nothing looks obviously broken with this change).

File: data/controller_i18n_links.yaml (hunk 1 above)
Comment (oraNod) [reply data/controller_i18n_links.yaml:3]: yeah we could create an easyfix issue for that. it was inherited from the names of the HTML pages on the docsite (e.g. `automation-tower-chinese-translations.html`) so I just made the key match the page. we can totally change that though.

**Pull request merged**